string autonomous_time              
string autonomous_receiver_status   
string autonomous_current_main      
string autonomous_current_main_state
string autonomous_current_util      
string autonomous_current_util_state
string autonomous_current_event     
string autonomous_current_event_time